civihr_employee_portal/views/views_export/views_emergency_contacts.inc

```diff
@@ -85,6 +85,22 @@ $handler->display->display_options['fields']['dependant_s__92']['field'] = 'depe
 $handler->display->display_options['fields']['dependant_s__92']['label'] = '';
 $handler->display->display_options['fields']['dependant_s__92']['exclude'] = TRUE;
 $handler->display->display_options['fields']['dependant_s__92']['element_label_colon'] = FALSE;
+/* Field: Global: Custom text */
+$handler->display->display_options['fields']['emergency_contact_deletion']['id'] = 'something';
+$handler->display->display_options['fields']['emergency_contact_deletion']['table'] = 'views';
+$handler->display->display_options['fields']['emergency_contact_deletion']['field'] = 'nothing';
+$handler->display->display_options['fields']['emergency_contact_deletion']['label'] = '';
+$handler->display->display_options['fields']['emergency_contact_deletion']['element_label_colon'] = FALSE;
+/* Contextual filter: CiviCRM Custom: Emergency Contacts: Entity ID */
+$handler->display->display_options['arguments']['entity_id']['id'] = 'entity_id';
+$handler->display->display_options['arguments']['entity_id']['table'] = 'civicrm_value_emergency_contacts_21';
+$handler->display->display_options['arguments']['entity_id']['field'] = 'entity_id';
+$handler->display->display_options['arguments']['entity_id']['default_action'] = 'default';
+$handler->display->display_options['arguments']['entity_id']['default_argument_type'] = 'php';
+$handler->display->display_options['arguments']['entity_id']['default_argument_options']['code'] = 'return \\CRM_Core_Session::getLoggedInContactID();';
+$handler->display->display_options['arguments']['entity_id']['summary']['number_of_records'] = '0';
```

Comment:
Do we really need the summary part for the for adding the arguments? Will the code still work if the section is  removed?

Reply:
Probably a lot of the auto exported stuff could be removed, but then it will likely be re-added the next time someone exports the view. Or do you mean there's some way to disable exporting this from the views UI?

Reply:
Not really, was just curious. We can leave as is then since its auto exported.